tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 1.6
-envlist = py3,py27,pypy,pep8
+envlist = py3,py27,pypy,lint
```

Comment:
drop 2.7 from this as well?

Reply:
I'll keep this for a follow up. I suspect there are other mentions on 2.7 that we can drop too